metadata-service/auth-impl/src/main/java/com/datahub/authorization/DataHubAuthorizer.java

```diff
@@ -184,15 +189,32 @@ private boolean isRequestGranted(final DataHubPolicyInfo policy, final Authoriza
     if (AuthorizationMode.ALLOW_ALL.equals(mode())) {
       return true;
     }
+
+    Optional<Urn> actorUrn = getUrnFromRequestActor(request.getActorUrn());
+    if (actorUrn.isEmpty()) {
+      return false;
+    }
+
+    final ResolvedResourceSpec actorResolvedResourceSpec = _resourceSpecResolver.resolve(
```

Comment:
Same question as above here - is there any way we can pass down the ResourceSpec and simple resolve once we need to?

Reply:
The same answer applies here.